.github/workflows/deploy.yml

```diff
@@ -27,6 +27,16 @@ jobs:
           cache: yarn
           cache-dependency-path: website/yarn.lock
 
+      - name: Cache code outputs
+        id: cache-code-outputs
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-code-outputs
+        with:
+          path: website/build/solutions
```

Comment:
is this compatible with https://github.com/microsoft/rise4fun/pull/29/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R12 ?